Projects/BricKuber/brickuber_lib.py

```diff
@@ -16,7 +16,20 @@
 import brickpi3 # import the BrickPi3 drivers
 import commands # import system command support
 
+debug_print_commands_on = True
```

Comment:
Do we want all the debug stuff in here? It adds "clutter" to the code and to the terminal.

Reply:
Turned it off.  Good catch.